util/types/json/serdes.go

```diff
@@ -224,8 +232,10 @@ func encodeJSONString(s string, buffer *bytes.Buffer) {
 	byteArray := hack.Slice(s)
 	var varIntBuf = make([]byte, 9)
 	var varIntLen = binary.PutUvarint(varIntBuf, uint64(len(byteArray)))
-	buffer.Write(varIntBuf[0:varIntLen])
-	buffer.Write(byteArray)
+	_, err := buffer.Write(varIntBuf[0:varIntLen])
+	terror.Log(err)
```

Comment:
Why not return error here?

Reply:
The function doesn't have an error return value.
Actually, the buffer never returns any error.
I could change all the functions to return an error, but that changes much more code, so I just log the error for the errors that we are sure never happens for now.

Later we can decide if it's worth to add the error return value.